src/components/rhelGraphCard/__tests__/rhelGraphCard.test.js

```diff
@@ -37,4 +40,38 @@ describe('RhelGraphCard Component', () => {
 
     expect(component).toMatchSnapshot('fulfilled');
   });
+
+  it('should render breakpoint states', () => {
+    const props = {
+      error: false,
+      pending: false,
+      fulfilled: true,
+      breakpoints,
+      currentBreakpoint: 'xs'
+    };
+    const component = shallow(<RhelGraphCard {...props} />);
+
+    expect(component).toMatchSnapshot('xs breakpoint');
+
+    component.setProps({
+      currentBreakpoint: 'sm'
+    });
+    expect(component).toMatchSnapshot('sm breakpoint');
+    component.setProps({
+      currentBreakpoint: 'md'
+    });
+    expect(component).toMatchSnapshot('md breakpoint');
+    component.setProps({
+      currentBreakpoint: 'lg'
+    });
+    expect(component).toMatchSnapshot('lg breakpoint');
+    component.setProps({
+      currentBreakpoint: 'xl'
+    });
+    expect(component).toMatchSnapshot('xl breakpoint');
+    component.setProps({
+      currentBreakpoint: 'xl2'
+    });
+    expect(component).toMatchSnapshot('xl2 breakpoint');
+  });
```

Comment:
comment:
Each app we work on does this different, advanced apologies =). Whitespace around code isn't a bad thing =) feel free, in the future, to add spacing in between those `expects` or even bunched up `if` conditions, like in the graphHelpers. It's related to being able to do a visual scan quickly